modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -714,7 +714,7 @@ MATCH ()-[r: KNOWS]->() WHERE r.since IS NOT NULL RETURN r
 =====
 
 [#_neo_clientnotification_statement_codegenerationfailed]
-=== CodeGenerationFailed
+=== Unsupported runtime - CodeGenerationFailed
```

Comment:
I suppose, this will be fixed. But for now we have two notifications with the same headings